INSTALL.md

```diff
@@ -26,10 +26,11 @@ The following packages you should be able to install from your package manager:
 - phpunit
 - nodejs
 - npm
+- bower (run `npm install -g bower` on the command line)
```

Comment:
We should mention, that it may be necessary to (soft) link the nodejs executable to `node`.

```bash
sudo ln -s /usr/bin/nodejs /usr/bin/node
```